.github/workflows/daily_test_version.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 Intel Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Publish commit for daily testing
+
+on:
+  schedule:
+    - cron:  '50 22 * * *'
+  push:
+    tags:
+    - '*'
```

Comment:
what does the '*' do?

Reply:
you tell me, I copied that from your west action :-)